scripts/manualpreview.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+# Check if jq is installed
+hash jq 2>/dev/null || { echo >&2 "Error: jq is not installed. Please install jq before running this script."; exit 1; }
+
+# Check if gh is installed
+hash gh 2>/dev/null || { echo >&2 "Error: GitHub CLI (gh) is not installed. Please install gh before running this script."; exit 1; }
+
+# Cehck if inside a repository
+if ! git rev-parse --is-inside-work-tree > /dev/null 2>&1; then
+  echo "ERROR: you must execute this script from the OpenShift Git repository." 1>&2
+  exit 1
+fi
+
+PR_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+PR_NUMBER="$(gh pr list -H $PR_BRANCH --json number | jq -r '.number')"
```

Comment:
https://www.shellcheck.net/wiki/SC2086

```suggestion
PR_NUMBER="$(gh pr list -H "$PR_BRANCH" --json number | jq -r '.number')"
```

Reply:
bash-lsp tells me to do this, but I can't get the script to run so :shrug: